### DISCUSSION TITLE
cells used in create Fragment object

### DISCUSSION THREAD

**Discussion by ashleshagogate** [stuart-lab/signac#1478] [Q&A] [answered]
Hello, 
While following the merging vignette, in the create fragment object function, can I include the cells in the barcodes.tsv file from the filtered_feature_bc_matrix folder output by cell ranger as the "cells" argument? This is because trying different parameters for filtering the cells from the metadata always gives me a "Not all cells requested could be found in the fragment file" error. 
Thanks!

**Comment by timoast** [comment, marked as answer]
Yes you could try that. If you are seeing "Not all cells requested could be found in the fragment file" you may also want to double check the formatting of cell barcodes in the fragment file and the count matrix
